.husky/.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
If there's no `.gitattributes` file, then the default git configuration on Windows will give CRLF line endings to these files. Unfortunately, that causes the hooks to fail because `sh` only accepts LF (even on Windows).

Explicitly specifying LF is only necessary for Windows, but it won't hurt other platforms.